twine/utils.py

```diff
@@ -295,3 +298,28 @@ def bool_from_env(val: Optional[str]) -> bool:
         """Allow '0' and 'false' and 'no' to be False."""
         falsey = {"0", "false", "no"}
         return bool(val and val.lower() not in falsey)
+
+
+_MAX_VERBOSITY = 5
+
+_VERBOSITY_TO_LOG_LEVEL = {
+    0: logging.WARNING,
+    1: logging.INFO,
+    2: logging.DEBUG,
+    3: _MAX_VERBOSITY,
+}
```

Comment:
Looking at the [flake8 implementation](https://github.com/PyCQA/flake8/blob/a7be77f761a4c29121d6bb6f61c11902281f9105/src/flake8/__init__.py#L27-L39), it seems like this is missing a `logging.addLevelName` call, as well as an explanation of why `_MAX_VERBOSITY` is 5.

However, again I wonder if we really need `_MAX_VERBOSITY` (or anything beyond `logging.INFO` for now).